AmazonCognitoAuthDemo/build.gradle

```diff
@@ -4,6 +4,7 @@ buildscript {
     repositories {
         jcenter()
         google()
+        mavenCentral()
```

Comment:
mavenCentral() repository isn't needed for the build scripts. Is there a specific reason you added this? Please remove if its not needed.